app/services/create_batch.rb

```diff
@@ -1,6 +1,6 @@
 class CreateBatch
   def call
-    unprocessed_appointment_summaries = AppointmentSummary.unprocessed
+    unprocessed_appointment_summaries = AppointmentSummary.unprocessed.where(format_preference: :standard)
```

Comment:
Should we pull out a scope here? This class is already tightly coupled to ActiveRecord in any case